populus/cli/chain_cmd.py

```diff
@@ -63,11 +63,12 @@ def chain_run(name, mine, verbosity, active):
     ensure_account_exists(data_dir)
 
     kwargs = {
-        "logfile": logfile_path,
-        "verbosity": "%d" % verbosity
+        # "logfile": logfile_path,
+        "verbosity": "%d" % verbosity,
+        "rpccorsdomain": 'http://localhost:5000'
```

Comment:
Have you tested this?  I believe the `http://` should be left off but I can't recall if that is the case or not.